### ISSUE TITLE
Support nested code blocks

### ISSUE THREAD

**Issue by astoicstradingtools** [nothingislost/obsidian-codemirror-options#64]
Hello,

Thank you for this plugin. It makes obsidian really look great in edit mode and reduces the switching.

I was wondering whether you could support nested code blocks? For example, I have Tasks search queries embedded in admonitions. The admonition renders well but not the search query.

`````
````ad-done
title: tasks to prioritize today
collapse: open
```tasks
sort by description
due on 2021-11-02
path does not include template
hide recurrence rule
hide due date
hide done date
```
````
`````

**Comment by astoicstradingtools** [comment]
I just realized the task search queries rendering is not yet enabled? :) https://github.com/schemar/obsidian-tasks/discussions/361
![image](https://user-images.githubusercontent.com/93439609/139835563-27e0d32c-17c5-46e4-aec0-8a8b42551787.png)

**Comment by nothingislost** [comment]
Yes, you're jumping the gun a bit :) but tasks will be supported soon.

This specific nested example probably won't be supported at release but I'll take a look at how difficult it would be to support.

**Comment by astoicstradingtools** [comment]
Oh my bad !! Indeed, I was very eager, too eager, to start using it haha.
Looking forward to the next releases. (I'll probably unnest in the meantime)

**Comment by astoicstradingtools** [comment]
A-wesome! Thank you 🙏

**Comment by nothingislost** [comment]
Just released in 0.7.0.
